regression/pages/ecommerce/back_to_basket_page.py

```diff
@@ -0,0 +1,53 @@
+"""
+Pages where user is landed after clicking back button on checkout page
+"""
+from bok_choy.page_object import PageObject
+
+from regression.pages.whitelabel.const import ECOMMERCE_URL_WITH_AUTH
+
+
+class BackToBasketPage(PageObject):
```

Comment:
I think you can refactor all the error message logic. Have one function get the error message, and have other functions get the various pieces of the error message that are important at that time. The error message could also be stored as a dict so that you can easily access the different parts of it for your tests.